apps/api/src/rest/routes/courses.ts

```diff
@@ -11,10 +12,11 @@ import {
   responseSchema,
 } from "$schema";
 import { CoursesService } from "$services";
-import { OpenAPIHono, createRoute } from "@hono/zod-openapi";
+import { OpenAPIHono, createRoute, z } from "@hono/zod-openapi";
```

Comment:
It has not been the norm to import and declare these schemas inline to REST; they are usually defined in the relevant schemas file and imported to REST. I understand it's simple here but in the future we may want, say, GraphQL to parse with this schema. This would mean GraphQL would import and depend on REST code, which doesn't make sense.